test/mocks/contexts/MockSignedInUserContext.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I see we're making mock contexts for the tests. I'm fine with this since it'll be useful in case multiple tests need the same context.